boilermaker/app.py

```diff
@@ -86,6 +90,12 @@ def register_async(self, fn, **options):
         logger.info(f"Registered background function fn={fn_name}")
         return self
 
+    def register_many_async(self, fns: list[TaskHandler], **options):
```

Comment:
nit: I think `register_all_async` might be a more natural name but don't ask my why. 1 letter shorter? English is weird so "many" != "all"? Case: "I ate many of the burritos" vs. "I ate all the burritos"